src/Components/benchmarkapps/Wasm.Performance/TestApp/Person.cs

```diff
@@ -15,7 +15,7 @@ public class Person
         public int Salary { get; set; }
         public bool IsAdmin { get; set; }
         public List<Person> Subordinates { get; set; }
-        public Dictionary<string, object> SecurityClearances { get; set; }
+        public Dictionary<string, bool> SecurityClearances { get; set; }
```

Comment:
I think it's fine to not support polymorphism to start with. The model here was slightly unnecessary. 

That said, it would be nice if the source generator could produce some visible warnings when it encounters these (you could add a compiler diagnostic when generating it) to say, this is unoptimized, consider doing something more useful.

Reply:
https://github.com/dotnet/runtime/issues/53935.